src/main/java/apprenticeship/Board.java

```diff
@@ -0,0 +1,28 @@
+package apprenticeship;
+
+import java.util.Arrays;
+
+public class Board {
+    public char cells[];
```

Comment:
If `cells` is public, then someone can change cells as they wish. For example, I could come in and write `cells[0] = "A"` in your array, even if the letter "A" is not one of the symbols you are currently using. It is better to make cells private and define two methods like `setCell` and `getCells` (does not have to be named like that, just a suggestion). Then, you can do some error checking in `setCell`. For example, if I try to set "A", the method might do nothing or return an error.